scripts/app.js

```diff
@@ -123,16 +127,22 @@ buttonZoomReset.addEventListener("click", function(event) {
 });
 
 function zoom(zoomValue) {
-    const currentFontSize = parseInt(window.getComputedStyle(input).fontSize);
+    const fontSizeRem = parseFloat(window.getComputedStyle(input).fontSize) / parseFloat(window.getComputedStyle(document.documentElement).fontSize);
+    const lineHeightRem = parseFloat(window.getComputedStyle(input).lineHeight) / parseFloat(window.getComputedStyle(document.documentElement).fontSize);
+    const outputLineHeightRem = parseFloat(window.getComputedStyle(output).lineHeight) / parseFloat(window.getComputedStyle(document.documentElement).fontSize);
```

Comment:
`parseFloat(window.getComputedStyle(document.documentElement).fontSize)` est répété souvent : tu peux l'extraire dans une constante.

`const documentFontSize = parseFloat(window.getComputedStyle(document.documentElement).fontSize);`

Reply:
I made the changes you suggested in the last commit.